examples/tts/radtts_train.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytorch_lightning as pl
+
+from nemo.collections.common.callbacks import LogEpochTimeCallback
+from nemo.collections.tts.models.radtts import RadTTSModel
+from nemo.core.config import hydra_runner
+from nemo.utils import logging
+from nemo.utils.exp_manager import exp_manager
+
+
+def freeze(model):
+    for p in model.parameters():
+        p.requires_grad = False
+
+
+def unfreeze(model):
+    for p in model.parameters():
+        p.requires_grad = True
```

Comment:
Please use NeMo's freeze and unfreeze functions: https://github.com/NVIDIA/NeMo/blob/f6987f4fb9e6c45afe0559e82d5d09e7d28a365d/nemo/core/classes/module.py#L49

Reply:
This implementation is a little different. it freeze during evaluation and unfreeze during training  which is not what I am doing.

Reply:
I thought the self.train was making all the modules in training state. I am using the methods from NeMo now. Thanks!